doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po

```diff
@@ -597,6 +615,9 @@ msgid ""
 "object is fresh enough to serve. If it is fresh, then Traffic Server serves "
 "it to the client as a *cache hit* (see the figure below)."
 msgstr ""
+"オブジェクトがキャッシュにある場合は、Traffic Serverはオブジェクトが提供する"
+"のに十分新鮮かどうかを確認します。新鮮な場合、Traffic Serverはそれを*キャッ"
+"シュヒット*としてクライアントに提供します（下図参照）。"
```

Comment:
Similar things are required by end-string of inline markup
> The end-string must end a text block (end of document or followed by a blank line) or be immediately followed by whitespace or any of ' " . , : ; ! ? - ) ] } / \ or >.
http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup